### PR TITLE
HSC-84: Enable multiple currencies

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

File: configuration/currency/res.currency.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+id,active
+"base.INR",false
```
Comment (rbuisson) [inline configuration/currency/res.currency.csv:2]: @nuhuhmutebi , unless I'm wrong, we don't need to disable the INR currency anymore.
This was activated by Bahmni in older version.
Using Odoo 14 with Ozone now should not enable INR.

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/currency/res.currency.csv (hunk 1 above)
Comment (nuhuhmutebi) [reply configuration/currency/res.currency.csv:2]: @rbuisson INR is actually set to false, unless you propose that we remove it completely

Comment (rbuisson) [reply configuration/currency/res.currency.csv:2]: I meant we can remove it from the file altogether and don't have to explicitely disable it.

**Comment by nuhuhmutebi** [comment]
Hi @rbuisson everything good now, we can go ahead and merge

File: configuration/setting/hsc_settings.xml [hunk 2]
```diff
@@ -4,7 +4,11 @@
         <field name="default_purchase_method">purchase</field>
         <field name="group_stock_production_lot" eval="True"/>
         <field name="group_lot_on_delivery_slip" eval="True"/>
+<<<<<<< HEAD
```
Comment (rbuisson) [inline configuration/setting/hsc_settings.xml:7]: Sounds like there is some fixing of conflicts that must be done.

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Pull request closed without merging**